examples/calculate_eventdisplay_sensitivity_cut2.py

```diff
@@ -0,0 +1,171 @@
+"""
+Example for using pyirf to calculate IRFS and sensitivity from EventDisplay DL2 fits
+files produced from the root output by this script:
+
+https://github.com/Eventdisplay/Converters/blob/master/DL2/generate_DL2_file.py
+"""
+import logging
+
+import numpy as np
+from astropy import table
+import astropy.units as u
+from astropy.io import fits
+import uproot
+
+from pyirf.io.eventdisplay import read_eventdisplay_fits
+from pyirf.binning import (
+    create_bins_per_decade,
+    add_overflow_bins,
+    create_histogram_table,
+    bin_center,
+)
+from pyirf.sensitivity import calculate_sensitivity, estimate_background
+from pyirf.utils import calculate_theta, calculate_source_fov_offset
+from pyirf.simulations import SimulatedEventsInfo
+
+from pyirf.spectral import (
+    calculate_event_weights,
+    PowerLaw,
+    CRAB_HEGRA,
+    IRFDOC_PROTON_SPECTRUM,
+    IRFDOC_ELECTRON_SPECTRUM,
+)
+
+
+log = logging.getLogger("pyirf")
+
+
+T_OBS = 50 * u.hour
+
+# scaling between on and off region.
+# Make off region 5 times larger than on region for better
+# background statistics
+ALPHA = 0.2
+
+# gh cut used for first calculation of the binned theta cuts
+INITIAL_GH_CUT = 0.0
+
+particles = {
+    "gamma": {
+        "file": "../data/event_display_dl2/gamma_onSource.S.3HB9-FD_ID0.eff-0_cuts2.fits",
+        "target_spectrum": CRAB_HEGRA,
+        "simulation_info": SimulatedEventsInfo(
+            n_showers=1986000000,
+            energy_min=0.003 * u.TeV,
+            energy_max=330.00 * u.TeV,
+            spectral_index=-2.0,
+            max_impact=2500.00 * u.m,
+            viewcone=0.0 * u.deg
+        )
+    },
+    "proton": {
+        "file": "../data/event_display_dl2/proton_onSource.S.3HB9-FD_ID0.eff-0_cuts2.fits.gz",
+        "target_spectrum": IRFDOC_PROTON_SPECTRUM,
+        "simulation_info": SimulatedEventsInfo(
+            n_showers=77388000000,
+            energy_min=0.004 * u.TeV,
+            energy_max=600.00 * u.TeV,
+            spectral_index=-2.0,
+            max_impact=3000.00 * u.m,
+            viewcone=10.0 * u.deg,
+        )
+    },
+    "electron": {
+        "file": "../data/event_display_dl2/electron_onSource.S.3HB9-FD_ID0.eff-0_cuts2.fits.gz",
+        "target_spectrum": IRFDOC_ELECTRON_SPECTRUM,
+        "simulation_info": SimulatedEventsInfo(
+            n_showers=3953200000,
+            energy_min=0.003 * u.TeV,
+            energy_max=330.00 * u.TeV,
+            spectral_index=-2.0,
+            max_impact=3000.00 * u.m,
+            viewcone=10.0 * u.deg)
+    },
+}
+
+
+def get_bg_cuts(cuts, alpha):
```

Comment:
Shouldn't this be aready in estimate_background?

Reply:
I didn't want to include this example add all, will remove